packages/plugins/minos-discovery-kong/minos/plugins/kong/discovery.py

```diff
@@ -50,7 +51,11 @@ def __init__(
         super().__init__(
             host, port, circuit_breaker_exceptions=(httpx.HTTPStatusError, *circuit_breaker_exceptions), **kwargs
         )
-        self.kong = KongClient(self.route)
+
+        if client is None:
+            client = KongClient()
```

Comment:
I think in this case it's better to propagate the `host` and `port` of the `KongDiscoveryClient` to the `KongClient`:
```suggestion
            client = KongClient(host=host, port=port)
```

Reply:
Done!